dll/win32/comctl32/propsheet.c

```diff
@@ -3580,6 +3580,27 @@ PROPSHEET_DialogProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
         }
         SetFocus(GetDlgItem(hwnd, IDOK));
       }
+#ifdef __REACTOS__
+      {/* try to fit it into the desktop */
```

Comment:
```suggestion
      { /* try to fit it into the desktop */
```
Add a spece before the comment.